package.json

```diff
@@ -11,11 +11,15 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
+    "@types/react-datepicker": "^4.1.7",
     "@types/react-dom": "^17.0.0",
     "apollo": "^2.33.6",
+    "classnames": "^2.3.1",
     "graphql": "^15.6.0",
     "msw": "^0.28.2",
+    "node-sass": "4.14.1",
```

Comment:
What required node-sass? I'm good with it, but would just like to understand why it is needed.